flutter/realm_flutter/tests/lib

```diff
@@ -1 +1 @@
-../lib
\ No newline at end of file
+../../../lib/
```

Comment:
I know this is not a new thing, but I'm curious why we need to symlink lib inside the tests app? AFAICT the test project takes a dependency on the `realm` package, so I would expect it to pull the lib source code from there.

Reply:
Indeed.. I just wanted to do the test refactoring in a separate PR as to not drown this one in small import updates.